src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-advanced-dashboarding.mdx

```diff
@@ -0,0 +1,471 @@
+
+import queriesnrql2tutorial9 from 'images/queries-nrql_screenshot-crop-tutorial2-9.webp'
+
+import queriesnrql2tutorial10 from 'images/queries-nrql_screenshot-crop-tutorial2-10.webp'
+
+import queriesnrql2tutorial11 from 'images/queries-nrql_screenshot-crop-tutorial2-11.webp'
+
+import queriesnrql2tutorial12 from 'images/queries-nrql_screenshot-crop-tutorial2-12.webp'
+
+import queriesnrql2tutorial13 from 'images/queries-nrql_screenshot-crop-tutorial2-13.webp'
+
+import queriesnrql2tutorial14 from 'images/queries-nrql_screenshot-crop-tutorial2-14.webp'
+
+import queriesnrql2tutorial15 from 'images/queries-nrql_screenshot-crop-tutorial2-15.webp'
+
+import queriesnrql2tutorial16 from 'images/queries-nrql_screenshot-crop-tutorial2-16.webp'
+
+import queriesnrql2tutorial17 from 'images/queries-nrql_screenshot-crop-tutorial2-17.webp'
+
+import queriesnrql2tutorial18 from 'images/queries-nrql_screenshot-crop-tutorial2-18.webp'
+
+import queriesnrql2tutorial19 from 'images/queries-nrql_screenshot-crop-tutorial2-19.webp'
+
+import queriesnrql2tutorial20 from 'images/queries-nrql_screenshot-crop-tutorial2-20.webp'
+
+We hope seeing example queries and explanations in the context of your own data helps you better understand how to transform data into powerful visuals. In this tutorial, you'll use what you learned in our previous tutorials on the [foundations of using NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/) and [controlling your data with NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-control-your-data/) and build on them with more advanced concepts.
+
+In this tutorial, you'll learn how to advance your dashbaording by using faceting by case, advanced aggregation functions, the `EXTRAPOLATE` keyword, filtering aggregation functions, and overriding values. Specifically, you'll learn how to use:
+
+* Advanced aggregation functions like `filter()`, `apdex()`, `rate()funnel()`, `histogram()`.
+* The `EXTRAPOLATE` clauses.
+* `FACET CASES()`, including how to use attribute and group matching values.
+* `filter()` to combine Event Types.
+* Overriding values, when necessary.
+
+You may not use these functions and features on every single dashboard, but they will certainly come in handy when tackling specific problems and requirements. Let's get started!
+
+### Use advanced aggregators 
+<Steps>
+    <Step>
+#### Calculating Rate
+Let's start with the `rate()` function. It allows you to visualize the frequency of events over time, which helps when you want to understand the frequency of events in small periods of time within larger time windows.
+
+In the example below, you can see the average frequency of requests every 5 minutes for the last hour compared to the previous hour's 5-minute average frequency. Notice the query uses `SINCE 1 hour ago`, this is the overall time window in which you're calculating the rate.
```

Comment:
```suggestion
In the example below, you can see the average frequency of requests every 5 minutes for the last hour compared to the previous hour's 5-minute average frequency. Notice the query uses `SINCE 1 hour ago`. This is the overall time window in which you're calculating the rate.
```

Reply:
Comma splice